js-pkg/react/src/main.tsx

```diff
@@ -64,16 +79,18 @@ export function usePresence<T extends Record<string, any>>(
     }
   }, [awareness])
 
-  const setLocalPresence = useCallback(
-    (localState: any) => {
-      if (awareness) {
-        awareness.setLocalState(localState)
-      }
-    },
-    [awareness],
-  )
+  return presence
+}
+
+export function usePresence<T extends Record<string, any>>(): [
```

Comment:
Minor: I kinda think it might be better to not have a `usePresence` hook at all. In the best case, devs can write one line instead of two:

```ts
const presence = useGetPresence()
const setPresence = usePresenceSetter()
```

In the worst case, they are subscribing to presence updates unnecessarily and their app slows way down and they don't know why, so they just blame it on y-sweet.

I don't think the upside is enough to offset the downside.

Reply:
My only hesitation on this is that I think that discourages people from specifying the generic type, since they have to keep it in sync between two locations. But that's a minor thing, they can always create a type alias.